other/astyle/astylerc

```diff
@@ -9,15 +10,19 @@
 
 # Padding Options
 --pad-header
---break-blocks
 --pad-oper
+#--unpad-brackets
 --unpad-paren
 --align-pointer=name
 --align-reference=name
+#--squeeze-lines=1
+#--squeeze-ws
```

Comment:
Why some of these are commented out? The astyle version on CI is too old and doesn't have those yet? Maybe leave a comment in the file explaining this as it's not very obvious?

What would you say about changing our style script to run in a Docker, so that all contributors would use the same astyle version, formatting the code the same way?

Reply:
https://github.com/TokTok/c-toxcore/pull/2634

Reply:
I have a commit with docker image formatter other/docker/format/run, but it's not ready yet. I'll eventually do it.

Reply:
(Just some unactionable story/explanation, not directed at @iphydf specifically.

I remember the formatter version being a big issue before when we tried using clang-format -- everyone had to use the exact (or newer?) version of clang-format as it errors on unknown options, when astyle just ignores them and proceeds formatting using only options it recognizes, so in contrast developers could use any version of astyle and the code would be formatted "good enough" even if it's an older version. But with the CI (the restyled check) enforcing exact formatting, "good enough" formatting no longer cuts it and it has to be exactly the same as what the CI expects, which is why the Docker container is now needed when it previously wasn't.)